pkg/inject/proxy.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 const (
+	defaultRunAsUser          = 0
```

Comment:
Why does proxy need to run as root?

Reply:
Although this is defined in proxy.go, the proxy config is only used in initProxyMutator and cniProxyMutator, and in cniProxyMutator, this field is not used so runAsUser is not set there.